.changeset/soft-horses-ring.md

```diff
@@ -0,0 +1,8 @@
+---
+"@comet/admin-theme": patch
+---
+
+Adapt `Radio` and `Checkbox` styling to Comet DXP design.
+
+Fix colors of disabled states.
+Remove `secondary` color styling.
```

Comment:
Isn't this a breaking change?

Reply:
I adapted the changeset 52b145149911f7400a455f7b0733c4a62dc0ccd8 and also changed the target branch to `next`

Reply:
@juliawegmayr could you maybe spilt this into two PRs (and two changesets)? 

- Fix the colors and remove the stories using the secondary color: `patch` in `main`
- Remove the custom theme-styling of the `secondary` variant: `major` in `next`

Reply:
Sure! 
This is the branch for the main branch: https://github.com/vivid-planet/comet/pull/2871
and this branch was also updated.

Reply:
Thanks 🙂 

Could you please update this PRs title and description to match what this PR is doing now.